app/src/main/java/nerd/tuxmobil/fahrplan/congress/favorites/StarredListFragment.kt

```diff
@@ -128,6 +130,8 @@ class StarredListFragment :
             val adapter = StarredListAdapter(activity, sessions, numDays, useDeviceTimeZone)
             currentListView.adapter = adapter
             activity.invalidateOptionsMenu()
+
+            spinnerView.visibility = View.GONE
```

Comment:
You can use the `isVisible` extension function here.

Did you try to remove this totally by just setting `android:visibility="gone"` in the layout?

Reply:
This is the point where the view gets hidden. By default the loading spinner is shown and it gets hidden when the first data arrives at the view model. I added a check to `isVisible` in order to only set the visibility once

Reply:
Here is an example usage: https://github.com/EventFahrplan/EventFahrplan/blob/c3b6203746b901bfa52bea3d24a6b9c99827089d/app/src/main/java/nerd/tuxmobil/fahrplan/congress/details/SessionDetailsFragment.kt#L208

Reply:
:+1: Thanks for the change.